pom.xml

```diff
@@ -23,48 +21,56 @@
     <java.version>11</java.version>
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
-    <maven-plugins.version>2.22.2</maven-plugins.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <maven-plugins.version>2.22.2</maven-plugins.version>
     <start-class>edu.tamu.cap.CapApplication</start-class>
-    <packaging>war</packaging>
+    <maven.packaging>war</maven.packaging>
   </properties>
 
-  <dependencies>
+  <packaging>${maven.packaging}</packaging>
 
+  <dependencies>
     <dependency>
       <groupId>edu.tamu.weaver</groupId>
       <artifactId>auth</artifactId>
-      <version>2.1.1-RC9</version>
+      <version>${project.parent.version}</version>
     </dependency>
 
     <dependency>
       <groupId>edu.tamu.weaver</groupId>
       <artifactId>token-provider</artifactId>
-      <version>2.1.1-RC9</version>
+      <version>${project.parent.version}</version>
     </dependency>
 
     <dependency>
       <groupId>edu.tamu.weaver</groupId>
       <artifactId>validation</artifactId>
-      <version>2.1.1-RC9</version>
+      <version>${project.parent.version}</version>
     </dependency>
 
     <dependency>
       <groupId>edu.tamu.weaver</groupId>
       <artifactId>wro</artifactId>
-      <version>2.1.1-RC9</version>
+      <version>${project.parent.version}</version>
+      <exclusions>
```

Comment:
This exclusion workaround is corrected in latest release of weaver and can be removed.